extension-manifest-v3/src/pages/settings/views/trackers.js

```diff
@@ -65,8 +65,15 @@ function clearCategory(id) {
             store.ready(t.exception) &&
             t.exception.blocked !== category.blockedByDefault,
         )
-        .map((tracker) =>
-          store.set(tracker.exception, { blocked: category.blockedByDefault }),
+        .map(({ exception }) =>
+          store.set(
```

Comment:
Much better, as when there is no exceptions, the whole model is removed, so in the case of being blocked by default, it does not introduce harmful filters.